code.cpp

```diff
@@ -0,0 +1,136 @@
+#include <array>
+#include <cmath>
+#include <cstdlib>
+#include <iostream>
+#include <unistd.h>
+#include <vector>
+
+const int length = 32;
+const int hight = 11; 
+const int tabl_size = length * hight;
+
+std::array<int,tabl_size> null_array() { 
+    std::array<int,tabl_size> null;
```

Comment:
<int, table_size>

Reply:
Also I''m sure that std::array has a constructor, which will allow to do the same, so this function can be removed